Marlin/src/module/settings.cpp

```diff
@@ -1831,15 +1793,12 @@ void MarlinSettings::postprocess() {
       // Heated Chamber PID
       //
       {
-        PID_t pid;
+        Serialization::PIDSS pid;
```

Comment:
I'm discomfited seeing double-colons creeping into the codebase. I'm not fond of them because it makes your code look like VisualBASIC, and it's me who will end up having to type them out all day.

Philosophically and stylistically speaking, will it really be better if Marlin code is modified so that everything is defined in nested namespaces with functions and variables increasingly prefixed by a hierarchy of serious-sounding namespace and class names?

Reply:
Ok, no problem, I'll remove the namespace here. 


> Philosophically and stylistically speaking, will it really be better if Marlin code is modified so that everything is defined in nested namespaces with functions and variables increasingly prefixed by a hierarchy of serious-sounding namespace and class names?

I can't answer the question here. It's up to the original code writer to decide how to separate the "modules" of her code. I'm used with namespace and double colons, but I perfectly understand it's bothersome to use. So, no, there's no theoretical or better reason to use namespaces.